### DISCUSSION TITLE
newton_loops in dsem_control

### DISCUSSION THREAD

**Discussion by Arnaud-Gruss** [James-Thorson-NOAA/dsem#10] [Announcements]
Regarding the first Vignette demo, when I try to run the following:
```R
fit = dsem( sem = sem,
  tsdata = tsdata,
  estimate_delta0 = TRUE,
  control = dsem_control( 
    quiet = TRUE,
    newton_loops = 0 ) )
```
The estimation will not work and the following error message appears:
```R
Error in dsem_control(quiet = TRUE, newton_loops = 0) :
  unused argument (newton_loops = 0)
```
Is this because you removed ```newton_loops``` from ```dsem_control```, in which case ```newton_loops = 0``` should be removed from the first Vignette demo?

**Comment by James-Thorson-NOAA** [comment]
can you confirm that you're on the main branch of the github and it's fully
updated?  Or where are you pulling the vignette from?  The Current vignette
is here and has the correct syntax:
https://james-thorson-noaa.github.io/dsem/articles/vignette.html

**Comment by Arnaud-Gruss** [comment]
I actually needed to reinstall dsem to have the latest version of the codes ... Sorry that I did not think about doing that before submitting this issue. So there is no issue and I can close it. Apologies once again for the oversight!

**Comment by James-Thorson-NOAA** [comment]
no worries.  I just moved it to Discussions and closing for now.
